src/reference/modules/administration/pages/troubleshooting.adoc

```diff
@@ -1,5 +1,75 @@
-= Troubleshooting agent-server communication issues
+= Troubleshooting
 
 == Inventory problem
 
-== Reporting problem
+When you see error reports about the inventory component or when your last inventory date
+in node details is too old (i.e. more than one day).
+
+Run `rudder agent inventory -i`:
+
+* If it gives an *Invalid inventory* error, this is likely a bug, please open an issue with the provided error message.
+* If it fails with a curl error:
+
+** Check if the IP seen by the server (or relay) is in the allowed network
+** Check network connectivity to port 443 on server (or relay), for example using `telnet rudder.example.com 443`
+** Run the given curl command after removing the `-s` or `--silent` argument to have a complete error
+
+* If it succeeds then the problem is probably on the root server. Check the web application logs in
+  `/var/log/rudder/webapp/TODAY.stderrout.log` for lines about your node's inventory:
+
+** If you have an *Invalid signature* message
+
+*** It could be because of a change of node id or agent key. Compare the certificate fingerprint and node id from the web interface and the `rudder agent info` output.
+*** There was a problem with the signature process on the node. Particularly, when there is a problem with "openssl" command, the signature may be empty. Check the content of signature file for that inventory in `/var/rudder/inventories/failed/xxx-yyy-zzz.ocs.sig-DATE`
+*** The node sending the inventory is not the same as the one registered in Rudder. This can show a security problem, but it may also be due to two nodes having the same nodeID for some reason (a VM cloned followed with an update of the agent key of one the cloned node, but no change of the node ID).
+In this case `rudder agent factory-reset` should be run after VM clone.
+
+** If you have an *Invalid inventory* message this is likely a bug, please open an issue with
+   the provided error message.
+** If you have no message related to your node
```

Comment:
i don't understand this sentence in this context

Reply:
we're still on the sublevel of "check webapp logs for messages related to your node's id" item, I'll make it more precise